internal/pipe/docker/docker_test.go

```diff
@@ -1002,6 +1002,18 @@ func TestDefaultDockerfile(t *testing.T) {
 	require.Equal(t, "Dockerfile", ctx.Config.Dockers[1].Dockerfile)
 }
 
+func TestDraftRelease(t *testing.T) {
+	var ctx = &context.Context{
+		Config: config.Project{
+			Release: config.Release{
+				Draft: true,
+				},
```

Comment:
ci will complain about lack of gofmp probably...

Reply:
@caarlos0 thanks! 👍 fixed